iso/Dockerfile

```diff
@@ -20,6 +20,10 @@ FROM boot2docker/boot2docker:1.12.3
 RUN echo "1.12.3" > $ROOTFS/etc/version
 ENV DOCKER_SHA a3bb88649bcc47775c29dab7d95709f4dda04d0e
 
+# Add the /etc/os-release file
+ADD os-release $ROOTFS/etc
+RUN sed -i '/os-release/,/^EOOS/d' /make_iso.sh
```

Comment:
I am confused, are we not generating the _os-release_ file as part of this pull request as well? If we control the file, why do I need to use _sed_ then?